src/utils/authentication.js

```diff
@@ -0,0 +1,20 @@
+import jwt from 'jsonwebtoken';
+
+export const generateToken = (payload, expiresIn = '24h') => jwt.sign(payload, process.env.JWT_SECRET, { expiresIn });
+
```

Comment:
when we generate a token, we want to call the `shuffleToken` method to scramble it before sending it.